plenum/server/config_req_handler.py

```diff
@@ -44,6 +44,9 @@ def validate(self, req: Request):
 
         typ = operation.get(TXN_TYPE)
         if typ == TXN_AUTHOR_AGREEMENT:
+            if self.state.get(self._state_path_taa_aml_latest()) is None:
+                raise InvalidClientRequest(identifier, req_id,
+                                           "TAA txn is forbidden until TAA AML isn't set. Send TAA AML first.")
```

Comment:
A correct spelling: `until TAA AML is set`

Reply:
fixed